package.json

```diff
@@ -5,10 +5,12 @@
   "main": "app.js",
   "type": "module",
   "scripts": {
-    "start": "npx eslint . && node app.js",
-    "build": "npx eslint . && babel . --ignore node_modules,es5-packaged,puppeteer --out-dir es5-packaged",
+    "start": "./node_modules/.bin/eslint . && node app.js",
+    "build": "./node_modules/.bin/eslint . && babel . --ignore node_modules,es5-packaged,puppeteer --out-dir es5-packaged",
```

Comment:
Thanks, this is a safer way to deal with packages in node_modules than the naive npx